backend/app/api/ws/chat_namespace.py

```
@@ -0,0 +1,1090 @@
+
+    Returns:
+        True if user can access the task
+    """
+    db = SessionLocal()
+    try:
+        task = (
+            db.query(Kind)
+            .filter(
+                Kind.id == task_id,
+                Kind.kind == "Task",
+                Kind.is_active == True,
```

Ruff (0.14.8)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)